ros2_batch_job/osx_batch/__init__.py

```diff
@@ -38,6 +38,13 @@ def pre(self):
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
+        if 'OPENSSL_ROOT_DIR' not in os.environ:
+            import subprocess
```

Comment:
My bad I thought it was narrowest scope, fixed in https://github.com/ros2/ci/pull/76/commits/01ec8ee5ee202ff9c7fb7e4fa3547ff653e5c628